Services/BrokerService/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BrokerService
+{
+    public static class Constants
+    {
+        internal static int Depth = 5;
+        public static string Token = "t.-lW-yXwRofmPWDrSqm9XpBO2K4WbTmHPkKPgRskRDlMqLCxzbP8efGmM3l6U1HU9ReMg5R0t3QqO86twfG3-vw";
```

Comment:
Mmmmmmm, pushed token to the public repository, nice

Reply:
> Mmmmmmm, pushed token to the public repository, nice

The token is also used by another public class at TinkoffBroker. But I still deleted this static class.